website/docs/developer/research/research-reconciliation.mdx

```diff
@@ -115,31 +115,41 @@ seemed a bit challenging and did not appear to provide a real benefit.
 
 #### Granule Metadata Object
 
-| Attribute Name   | Data Type | Description                                                            | Notable Items   |
-| ---------------- | --------- | ---------------------------------------------------------------------- | --------------- |
-| cumulus_granule_id | `string`  | granuleId that is provided by Cumulus.                                 | Must be unique per collectionId |
-| acquisition_time | `timestamp` | Date and time in UTC that the data was captured.                     | TBD - Not sure if we can get this from Cumulus. Potentially Optional but would like to use as date comparator.|
-| ingest_time      | `timestamp` | Date and time in UTC that the data was originally ingested into ORCA | None |
-| last_update      | `timestamp` | Date and time in UTC that information was updated.                   | Potential time filter for Cumulus reconciliation. |
-| archive_location | `string`  | S3 bucket the data was archived to.                                    | Potentially optional as this could be stored at the file level but would be duplicated. |
+| Attribute Name      | Data Type   | Description                                                          | Notable Items                            |
+| ------------------- | ----------- | -------------------------------------------------------------------- | ---------------------------------------- |
+| id                  | `string`    | Internal ORCA granule ID pseudo key.                                 | None                                     |
+| collection_id       | `string`    | Collection ID from Cumulus that refrences the Collections table.     | None                                     |
+| cumulus_granule_id  | `string`    | granuleId that is provided by Cumulus.                               | Must be unique per collectionId          |
+| execution_id        | `string`    | Step function execution ID from AWS.                                 | Unique ID automatically generated by AWS |
+| ingest_time         | `timestamp` | Date and time in UTC that the data was originally ingested into ORCA | None                                     |
+| cumulus_createdAt   | `timestamp` | time filter for Cumulus reconciliation.                              | None                                     |
```

Comment:
Would prefer to describe what this data represents rather than just its usage if possible.

Reply:
is this acceptable?
`Cumulus timestamp as part of the sync granule task output for reconciliation`
or do you have any other suggestions?

Reply:
Works for me.

Reply:
I concur, Use the same as the description in the table. See notes on the description and name. I would also put it in alphabetical order in the table.